HRIS.Services/Interfaces/IEmployeeTypeService.cs

```diff
@@ -9,19 +9,19 @@ public interface IEmployeeTypeService
     /// </summary>
     /// <param name="employeeTypeDto"></param>
     /// <returns></returns>
-    Task<EmployeeTypeDto> SaveEmployeeType(EmployeeTypeDto employeeTypeDto);
+    Task<EmployeeTypeDto> CreateEmployeeType(EmployeeTypeDto employeeTypeDto);
 
     /// <summary>
     ///     Delete Employee Type
     /// </summary>
     /// <param name="name"></param>
     /// <returns></returns>
-    Task<EmployeeTypeDto> DeleteEmployeeType(string name);
+    Task<EmployeeTypeDto> DeleteEmployeeType(int Id);
 
     /// <summary>
     ///     Get Employee Type
     /// </summary>
-    /// <param name="name"></param>
+    /// <param Id="Id"></param>
     /// <returns></returns>
     Task<EmployeeTypeDto> GetEmployeeType(string name);
```

Comment:
```suggestion
    Task<EmployeeTypeDto> GetEmployeeTypeByName(string name);
```

Reply:
The method needs to be more description on gets.

`GetEmployeeType` does not describe how to get employ, hence something like `GetEmployeeTypeByName` fits the actions of the method better.